### DISCUSSION TITLE
Textadept crashing (segfault)

### DISCUSSION THREAD

**Discussion by bloodofdracula** [orbitalquark/textadept#495] [General]
Moved the standard init.lua as supplied after installing from scratch (cmake and so on) to my ~/.textadept directory

Restart textadept-gtk (GUI)

Zorin Pro 16.3 distribution GTK

Multiple segfault.

Same if I simply copy init.lua

**Comment by orbitalquark** [comment]
If you're installing from scratch, then please do not copy any *init.lua* to your *~/.textadept/* folder. Textadept will automatically create a blank *init.lua* for you.
